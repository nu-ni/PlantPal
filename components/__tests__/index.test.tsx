// Nino Snapshot Test
import React from 'react';
import { render } from '@testing-library/react-native';
import Index from '@/app/(tabs)';


describe('Index Component', () => {
  it('matches the snapshot', () => {
    const tree = render(<Index />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
