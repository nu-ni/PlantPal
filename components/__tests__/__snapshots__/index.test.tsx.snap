// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Component matches the snapshot 1`] = `
<View
  style={
    {
      "backgroundColor": "white",
      "flex": 1,
    }
  }
>
  <RCTScrollView
    contentContainerStyle={
      {
        "alignItems": "center",
        "paddingBottom": 50,
        "paddingTop": 100,
      }
    }
  >
    <View>
      <View
        style={
          {
            "alignItems": "center",
            "borderRadius": 250,
            "height": 500,
            "justifyContent": "center",
            "overflow": "hidden",
            "position": "absolute",
            "top": -200,
            "width": 500,
            "zIndex": -1,
          }
        }
      >
        <Image
          source={1}
          style={
            {
              "height": "150%",
              "resizeMode": "contain",
              "width": "150%",
            }
          }
        />
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "justifyContent": "center",
            "marginBottom": 30,
            "marginTop": 170,
          }
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "alignItems": "center",
              "backgroundColor": "#557F60",
              "borderRadius": 50,
              "height": 100,
              "justifyContent": "center",
              "width": 100,
            }
          }
        >
          <Text
            style={
              {
                "color": "white",
                "fontSize": 50,
                "marginBottom": 5,
                "textAlign": "center",
              }
            }
          >
            +
          </Text>
        </View>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "flexDirection": "row",
            "marginVertical": 20,
          }
        }
      >
        <Text
          style={
            {
              "fontSize": 24,
              "fontWeight": "bold",
              "marginRight": 10,
            }
          }
        >
          Welcome!
        </Text>
        <View
          collapsable={false}
          jestAnimatedStyle={
            {
              "value": {
                "transform": [
                  {
                    "rotate": "0deg",
                  },
                ],
              },
            }
          }
          style={
            {
              "transform": [
                {
                  "rotate": "0deg",
                },
              ],
            }
          }
        >
          <Text
            style={
              [
                {
                  "color": "#11181C",
                },
                {
                  "fontSize": 16,
                  "lineHeight": 24,
                },
                undefined,
                undefined,
                undefined,
                undefined,
                {
                  "fontSize": 28,
                  "lineHeight": 32,
                  "marginTop": -6,
                },
              ]
            }
          >
            ðŸ‘‹
          </Text>
        </View>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "marginBottom": 20,
            "paddingHorizontal": 20,
          }
        }
      >
        <Text
          style={
            {
              "fontSize": 16,
              "marginBottom": 10,
              "textAlign": "center",
            }
          }
        >
          Erstelle schnell Pflanzenprofile, erstelle individuelle GiessplÃ¤ne und teile sie mit Freunden oder Nachbarn. So bleiben deine grÃ¼nen Mitbewohner immer perfekt versorgt â€“ ganz ohne Stress!
        </Text>
        <Text
          style={
            {
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            }
          }
        >
          Mach Pflanzenpflege smart und einfach â€“ mit PlantPals!
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "marginBottom": 50,
            "marginTop": 30,
          }
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "alignItems": "center",
              "backgroundColor": "#557F60",
              "borderRadius": 60,
              "justifyContent": "center",
              "marginTop": 20,
              "padding": 25,
              "width": 200,
            }
          }
        >
          <Text
            style={
              {
                "color": "white",
                "fontSize": 18,
              }
            }
          >
            Import
          </Text>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
